src/openforms/config/migrations/0030_piwik_pro_support_squashed_0037_globalconfiguration_form_upload_default_file_types.py

```diff
@@ -12,149 +12,6 @@
 import openforms.config.models
 import openforms.emails.validators
 import openforms.template.validators
-from openforms.analytics_tools.constants import AnalyticsTools
-from openforms.analytics_tools.utils import (
-    get_cookies,
-    get_csp,
-    get_domain_hash,
-    update_analytical_cookies,
-    update_csp,
-)
-
-# Functions from the following migrations need manual copying.
-# Move them and any dependencies into this file, then update the
-# RunPython operations to refer to the local versions:
-
-
-def move_analytics_config(apps, _):
```

Comment:
this datamigration has executed during the 2.0 -> 2.1 upgrade, and upgrading to 2.2 from 2.0 is blocked so we can safely remove this